src/supertux/sector_parser.cpp

```diff
@@ -340,49 +342,66 @@ SectorParser::create_sector()
   if (!m_sector.in_worldmap())
   {
     auto& background = m_sector.add<Background>();
-    background.set_image(DEFAULT_BG);
-    background.set_speed(0.5);
-
-    auto& bkgrd = m_sector.add<TileMap>(m_sector.get_tileset());
-    bkgrd.resize(100, 35);
-    bkgrd.set_layer(-100);
-    bkgrd.set_solid(false);
-
-    auto& frgrd = m_sector.add<TileMap>(m_sector.get_tileset());
-    frgrd.resize(100, 35);
-    frgrd.set_layer(100);
-    frgrd.set_solid(false);
+    background.set_images(DEFAULT_BG_TOP, DEFAULT_BG, DEFAULT_BG_BOTTOM);
+    background.set_speed(0.3);
+    background.set_name("Snowy Hills");
```

Comment:
Having spaces in the name is quite bad when trying to reference the background in Squirrel.